### PR TITLE
Change the font box to QFontComboBox as per issue 238

### LINKED ISSUE #238: Show font name in the font itself
It's very confusing which font to select since there's no preview available during selection.
I hit this bug when I selected some unreadable font by mistake and I couldn't read or select anything further.

Something like this would be really very awesome:
image

(that is, showing the font name in the font itself :))

### PR DESCRIPTION
**Pull request by LorenDB**
This PR dedicated to @deepbluev7 for the major help with libs, cmake, and limited Wifi.

fixes #238

### CODE REVIEW

File: src/UserSettingsPage.cpp [hunk 1]
```diff
@@ -477,9 +478,9 @@ UserSettingsPage::UserSettingsPage(QSharedPointer<UserSettings> settings, QWidge
 
         QFontDatabase fontDb;
         auto fontFamilies = fontDb.families();
-        for (const auto &family : fontFamilies) {
-                fontSelectionCombo_->addItem(family);
-        }
+        // for (const auto &family : fontFamilies) {
+        //         fontSelectionCombo_->addItem(family);
+        // }
```
Comment (utkarsh2102) [inline src/UserSettingsPage.cpp:483]: I feel it's best to remove the lines of code that are not needed instead of commenting out?

**Review by utkarsh2102 — CHANGES_REQUESTED** [review]
Hello,

Thanks a ton for working on this! ❤️ 
However, could you please attach a screenshot or something of the list of fonts that are displayed after building nheko on top of this PR?

Besides, I do have a minor suggestion comment below and there is also a merge conflict 😄

**Comment by LorenDB** [comment]
Yeah, I will remove the comments and squash the commit. Screenshot attached.

**Comment by utkarsh2102** [comment]
In that case, this looks awesome! ❤️

**Comment by LorenDB** [comment]
All errors/weird diffs/etc. have been resolved. Ready to merge AFAIK.

**Comment by deepbluev7** [comment]
Thank you, awesome work!

**Comment by LorenDB** [comment]
> Thank you, awesome work!

Thank **you** too :D

**Pull request merged**